robots:
  mur620a:
    spray_distance:
      updated_at: '2025-11-28 14:48:07'
      value: 0.626
  mur620c:
    driver_nodes:
    - /mur620c/UR10_r/UR10_r_enable
    - /mur620c/UR10_r/controller_stopper
    - /mur620c/UR10_r/ewellix_tlt_node_r
    - /mur620c/UR10_r/gripper_interface
    - /mur620c/UR10_r/pose_composer
    - /mur620c/UR10_r/ros_control_controller_spawner
    - /mur620c/UR10_r/ros_control_stopped_spawner
    - /mur620c/UR10_r/rosnode_killer
    - /mur620c/UR10_r/ur_calibrated_pose_pub_node
    - /mur620c/UR10_r/ur_hardware_interface
    - /mur620c/UR10_r/ur_hardware_interface/ur_robot_state_helper
    - /mur620c/UR10_r/ur_twist_limiter
    - /mur620c/b_rep117_laser_filter
    - /mur620c/bms_can_node
    - /mur620c/f_rep117_laser_filter
    - /mur620c/fake_mir_joint_publisher
    - /mur620c/light_cmd_transcode
    - /mur620c/light_cmd_transcode_local
    - /mur620c/mir_battery_state_publisher
    - /mur620c/mir_bridge
    - /mur620c/mir_pose_simple
    - /mur620c/move_base_node
    - /mur620c/move_group
    - /mur620c/remove_tf_frames
    - /mur620c/rgb_control
    - /mur620c/robot_state_publisher
    - /mur620c/tf_remove_state_publisher_frames
    spray_distance:
      updated_at: '2025-11-28 15:38:13'
      value: 0.631
    updated_at: '2025-11-28 14:34:07'
servo_targets:
  left: 19.0
  right: 100.0
  updated_at: '2025-11-28 15:56:09'
spray_distance:
  updated_at: '2025-11-28 15:38:13'
  value: 0.631
